generic/tests/vlan.py

```diff
@@ -218,6 +246,42 @@ def flood_ping(src, dst):
             nc_transfer(test, 0, 1)
             nc_transfer(test, 1, 0)
 
+            if vlan == int(bridge_vlan):
+                error_context.context("Run netperf stress test between guests", logging.info)
+                utils_test.run_virt_sub_test(test, params, env, sub_type)
+                if vm.is_dead():
+                    vm.create(params=params)
```

Comment:
Is this an intend error? Which vm is this `vm`?
Can you explain why you create vm when vm is dead?

Reply:
No. Discussed with @xiywang , delete this statements.  It will raise error when sub_test `netperf_stress` fail.